src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -1031,6 +1090,93 @@ define([
             ]);
         },
 
+        getPreviewDocument: function() {
+            if (!!this.previewWindow) {
+                return this.previewWindow.document;
+            } else {
+                return this.sandbox.dom.find('iframe', this.$preview).contents()[0];
+            }
+        },
+
+        handleChanges: function(changes) {
+            if (!!changes.reload) {
+                this.getPreviewDocument().location.reload();
+            } else {
+                // foreach property which was changed
+                for (var propertyName in changes) {
+                    if (changes.hasOwnProperty(propertyName)) {
+                        if (-1 !== propertyName.indexOf(',')) {
+                            this.handleSequence(propertyName, changes[propertyName]);
+                        } else {
+                            this.handleSingle(propertyName, changes[propertyName]);
+                        }
+                    }
+                }
+            }
+        },
+
+        handleSequence: function(propertyName, content) {
+            var sequence = propertyName.split(','),
+                filter = '',
+                item, before = 0;
+
+            for (item in sequence) {
+                // check of integer
+                if (!/^\+?(0|[1-9]\d*)$/.test(sequence[item])) {
```

Comment:
And you could probably extract this regex to an own variable, you are using it more often. If such a complicated regex is necessary at all ;-)